LandmarkRegistration.py

```diff
@@ -747,12 +746,12 @@ def addFiducial(self,name,position=(0,0,0),associatedNode=None):
             position[i] = rasBounds[2*i]
           if position[i] > rasBounds[2*i+1]:
             position[i] = rasBounds[2*i+1]
-      fiducialList.AddFiducial(*position)
-      fiducialIndex = fiducialList.GetNumberOfFiducials()-1
+      fiducialList.AddControlPoint(vtk.vtkVector3d(*position), "")
```

Comment:
@jamesobutler Seeing this code, I'm wondering if we should change `AddControlPoint`, `AddNControlPoints`, `AddControlPointWorld`, `InsertControlPointWorld`, `InsertControlPoint` methods signature to take `double[3]` instead of this exotic `vtkVector3d`. For return value we don't really have any other safe option than vtkVector3d, but for input values double[3] would translate to a more pythonic interface.

Reply:
Yes I agree. From python it would be better to not have to require `vtkVector3d` here.

Reply:
Would you have time to make this change in Slicer core? I' still busy with processing all the queued up pull requests.

Reply:
Sure. Change the constructors to take `double[3]` type or add additional constructors with `double[3]`?

Reply:
Good point. Adding double[3] versions of the same methods would be an easy fix, so I think we could do that now. Do you think there is a solution for automatic conversion that would work for both C++ and Python?

Reply:
That change sounds good.  I guess we can hold off on this PR until that is done.

Reply:
After attempting this work regarding the new methods, I've been unsuccessful, so I won't be able to complete it.

Reply:
I've added the simpler control point get/set methods (that can directly return vector or take an input vector) in https://github.com/Slicer/Slicer/pull/6041.

Reply:
I've updated this to the new method that doesn't require a vtk.vtkVector3d input object. I've also gone further and updated variables and comments to the point list terminology instead of fiducial.